www/%username/wallet/payout/%back_to.spt

```diff
@@ -120,10 +120,10 @@ title = _("Withdrawing Money")
 
     % if exchange
         <div class="alert alert-{{ 'success' if success else 'danger' }}">{{
-            _("{0} have been taken from your Liberapay wallet. "
+            _("{0} has been taken from your Liberapay wallet. "
               "If the transfer is successful {1} will land in your bank account and {2} will be paid in fees.",
               -exchange.amount + exchange.fee, -exchange.amount, exchange.fee) if success else
-            _("The attempt to send {0} to your bank account has failed. Error message: {1}", -exchange.amount, exchange.note)
+            _("Could not send {0} to your bank account: {1}", -exchange.amount, exchange.note)
```

Comment:
The user is already making the attempt. Latter sentence can be avoided wholesale.
It is not a log, so nobody is grepping for "fail"